File: Game/server/Game/Equip/Equip.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Game.Equip
{
    public class Equip : Game.Item.Item
    {

        public Equip(Game.Bag.BItem bItem, BEquipExtra extra) : base(bItem)
        {

        }

        public override void CalculateFighter(Game.Fight.Fighter fighter)
        {
            fighter.Bean.Attack += 20.0f;
            fighter.Bean.Defence += 20.0f;
        }

        public override bool Use()
        {
            return false;
        }
    }
}

File: ZezexJava/server/src/Game/Equip/Equip.java
package Game.Equip;

import Game.*;

public class Equip extends Game.Item.Item {

	public Equip(Game.Bag.BItem bItem, BEquipExtra extra) {
		super(bItem);

	}

	@Override
	public void CalculateFighter(Game.Fight.Fighter fighter) {
		fighter.getBean().Attack += 20.0f;
		fighter.getBean().Defence += 20.0f;
	}

	@Override
	public boolean Use() {
		return false;
	}
}